CenterForOpenScience/osf.io
# [PLAT-1311] Expose GDPR user deletion for admin app

## Purpose

If we want to do a GDPR deletion we have to do it manually, this PR allows an admin to to do a GDPR delete without running a management command. 

## Changes

- adds GDPR deletion view to user detail page for admin app.
- fixs a bug where block display form broken button view.

## QA Notes

Delete Button:
![screen shot 2019-01-16 at 2 32 10 pm](https://user-images.githubusercontent.com/9688518/51274069-3669b280-199c-11e9-907b-06d5dabcd101.png)
Delete modal:
![screen shot 2019-01-16 at 12 23 34 pm](https://user-images.githubusercontent.com/9688518/51274070-3669b280-199c-11e9-8cd6-9f61eecf4c32.png)

Click the GDPR delete button for a user and you should get messages communicating whether the deletion was successful or what prevented it from deletion.

## Documentation

Internal use only, no docs.

## Side Effects

None that I know of.

## Ticket

https://openscience.atlassian.net/browse/PLAT-1311
